snippets/man_gen/hubl_custom_module_fields.json

```diff
@@ -0,0 +1,519 @@
+{
+  "Field Group":{
+    "prefix": "group.group",
+    "body": [
+      "{",
+        "\"id\": \"\",",
+        "\"name\": \"field_group\",",
+        "\"label\": \"Field group\",",
+        "\"sortable\": false,",
+        "\"required\": false,",
+        "\"locked\": false,",
+        "\"hierarchical\": false,",
+        "\"max_depth\": 0,",
+        "\"children\": [],",
+        "\"type\": \"group\",",
+        "\"default\": {}, }"              
+    ],
+    "description": "Field Group"
+  },
+  "Repeater":{
+    "prefix": "group.repeater",
+    "body":[
+      "{",
+      "  \"id\": \"\",",
+      "  \"name\": \"field_group\",",
+      "  \"label\": \"Field group\",",
+      "  \"sortable\": false,",
+      "  \"required\": false,",
+      "  \"locked\": false,",
+      "  \"occurrence\": {",
+      "    \"min\": null,",
+      "    \"max\": null,",
+      "    \"sorting_label_field\": null,",
+      "    \"default\": null",
+      "  },",
+      "  \"hierarchical\": false,",
+      "  \"max_depth\": 0,",
+      "  \"children\": [],",
+      "  \"type\": \"group\",",
+      "  \"default\": {} }"
+      
+    ],
+    "description": "Repeater Group"
+  },
+  "Date Field": {
+    "prefix": "field.date",
+    "body": [
+      "{",
+      "\"id\": \" \",",
+      "\"name\": \"${1:date_field}\",",
```

Comment:
Do you think it is worth having placeholders for all of the names/labels as you do here?

Reply:
Personally, yes.  For fools like me, it is helpful to see this.  I can look at the design manager and see the labels and know what did what in the code. I do think it worth adding the cursor placement to all.  you will note that i started on the first, just didn't make it beyond that :)